### DISCUSSION TITLE
Input elements are not defined

### DISCUSSION THREAD

**Discussion by Hizuvi** [tttapa/Control-Surface#540] [Q&A]
I'm trying to add some LED to a simple controller, but I can't get the input elements to work (NoteLED and others). Even following the examples I cannot get it to work.
Here is the code and error message:
![image](https://user-images.githubusercontent.com/64695616/128600697-2ac2110b-07f7-48c2-8778-6d24fe01e158.png)

I'm using an arduino uno and the official IDE, I imported the library using Sketch > Include Library > Add .ZIP library. The zip I used is the master branch.

**Comment by tttapa** [comment]
You might not be using the latest version of the master branch. It compiles correctly for me, and it also works in the CI: https://github.com/tttapa/Control-Surface/actions/workflows/examples.yaml 

Make sure that you do not have any old versions of the library installed, and then download and install the latest `master` version. If you enable verbosity in the IDE preferences, the IDE tells you which version of the libraries its using, and where they are installed.

Please post the actual code as text instead of images of text (use the `<>` button to format it), I cannot reproduce any errors using pictures of code.

~~~md
```cpp
// your code here
```
~~~
